core/src/modules/configLoaders/app.ts

```diff
@@ -22,7 +22,10 @@ export async function loadApp(
   let app = await App.scope(null).findOne({
     where: {
       id: configObject.id,
-      locked: getCodeConfigLockKey(),
+      [Op.or]: {
+        locked: getCodeConfigLockKey(),
+        state: "deleted",
+      },
```

Comment:
Question - we made the choice a while back that code-config'd models which were deleted become unlocked.  This is so the destroy tasks could eventually delete them.  Just checking that that was still the right choice.